pubsub/google/cloud/pubsub_v1/subscriber/client.py

```diff
@@ -129,4 +129,7 @@ def subscribe(self, subscription, callback=None, flow_control=()):
         subscr = self._policy_class(self, subscription, flow_control)
         if callable(callback):
             subscr.open(callback)
+        elif callback is not None:
```

Comment:
Sorry I missed this, but you need to add a `Raises:` section to this docstring

Reply:
My fault for forgetting.  :)
Added.  Squash and merge on green?